### PR TITLE
Cherry-pick callhome

### PR DESCRIPTION
**Pull request by niladrih**
This PR cherry-picks the following commits from the `develop` branch to the `release/2.0` branch:
- 5f93adc08974dcbaa3a06722066baa258eb94e43
- 27ab31ecbdcc63005e2d934966eecac2609ef139
- e2b21c914999ebf8d485274f839a611afb25ff6b
- 4e5908b207c626eed43299aa4080e5b29d387045
- 4dae614ec07a539632a07cec79b13906c346075c
- 906e970a055237fa7883845adee213e3305d838e

### CODE REVIEW

**Comment by niladrih** [comment]
bors try

File: scripts/release.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ SCRIPTDIR=$(dirname "$0")
 TAG=`get_tag`
 BRANCH=`git rev-parse --abbrev-ref HEAD`
 IMAGES=
-DEFAULT_IMAGES="exporters.metrics.pool"
+DEFAULT_IMAGES="exporters.metrics.pool operators.upgrade obs.callhome"
```
Comment (tiagolobocastro) [inline scripts/release.sh:53]: Please remove the upgrade, it's not for 2.0 release.

**Review by tiagolobocastro — CHANGES_REQUESTED** [review]

File: scripts/release.sh (hunk 1 above)
Comment (niladrih) [reply scripts/release.sh]: Removed. Good catch!

**Review by tiagolobocastro — APPROVED** [review]

**Comment by niladrih** [comment]
This PR depends on #37

**Comment by niladrih** [comment]
bors try

**Comment by niladrih** [comment]
bors try

**Comment by niladrih** [comment]
bors merge

**Comment by niladrih** [comment]
bors merge

**Comment by niladrih** [comment]
bors try

**Review by datacore-vkumar — APPROVED** [review]

**Comment by niladrih** [comment]
@GlennBullingham, @tiagolobocastro, @blaisedias -- I need help with merging

**Comment by GlennBullingham** [comment]
bors merge

**Pull request merged**